packages/zkdb/src/sdk/global/zkdatabase-client.ts

```diff
@@ -1,11 +1,13 @@
-import { ApiClient, getNetworkEnvironment, IApiClient } from '@zkdb/api';
+import { ApiClient, IApiClient } from '@zkdb/api';
 import { NetworkId, PrivateKey } from 'o1js';
 import { isBrowser, isNetwork } from '@utils';
 import { Authenticator } from '../authentication';
 import { ZKDatabaseImpl, ZKSystemImpl } from '../impl';
 import { ZKDatabase, ZKSystem } from '../interfaces';
 import { AuroWalletSigner, NodeSigner, Signer } from '../signer';
 import InMemoryStorage from '../storage/memory';
+import { database } from '@zkdb/api/src/graphql/database';
+import { ENetworkId } from '@zkdb/common/build/src';
```

Comment:
I think we need to export index.ts for  `database`
```suggestion
import { database } from '@zkdb/api';
import { ENetworkId } from '@zkdb/common';
```

Reply:
Fixed